lib/scrabble.rb

```diff
@@ -15,4 +27,27 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def prepare(word)
+    if word.class != Array
+      word.upcase!
+      word = word.chars
+    else
+      word
+    end
+  end
+
+  def return_score(word)
+    word.each do |letter|
+      points_array << point_values[letter].to_i
+    end
+    return points_array
+  end
+
+  def combine_score(points_array)
+    points_array.each do |point|
+      @points += point
+    end
```

Comment:
Like that you figured out how to do this with `#each`, but probably a good use case for `reduce`. Ruby 2.4 also gives us a `#sum` method that we can use on arrays.